experiments/20160805-mutagenesis/20160902-mutagenesis-pb2.md

```diff
@@ -0,0 +1,38 @@
+## Goals
+
+- generate mutant PB2 PCR products with full DNA fragment
```

Comment:
Almost there - this time round I think your goal was to test whether the primers work together or not, using MyTaq red as a proxy for Agilent's enzyme right?